Providers/Pocket/Model/PocketFeedStory.swift

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+/*
+ The Pocket class is used to fetch stories from the Pocked API.
+ Right now this only supports the global feed
+
+ For a sample feed item check ClientTests/pocketglobalfeed.json
+ */
+
+struct PocketFeedStory {
```

Comment:
I know it was like that before, but would it be possible to use Codable protocol here instead?